### ISSUE TITLE
[CAL-2362] implement cal.com embeds in formbricks.com

### ISSUE THREAD

**Issue by PeerRich** [calcom/cal.com#10811]
open a PR in: https://github.com/formbricks/formbricks, not here

- use our react embed
- create a component in formbricks that requires a cal.com link
- show the booking embed inside the survey

<sub>[CAL-2362](https://linear.app/calcom/issue/CAL-2362/implement-calcom-embeds-in-formbrickscom)</sub>

**Comment by suyash5053** [comment]
Can you please give more brief, actually Its not clear that how you want to use cal.com embeds, I spent many hours and have tried many things but i'm still confused.

**Comment by mattinannt** [comment]
Matti from Formbricks here 👋 Thanks @PeerRich for initiating this collab 😅

We are currently in a big update of our question types which should be finished beginning of next week. https://github.com/formbricks/formbricks/pull/698

After this is merged, we can start with this issue. Here are the requirements from our side:
* Create a new Question Type `Booking with cal.com`
* The Form for this Question Type in the Survey Editor should only have one field to ask for the cal.com username (`calLink`)
* Integrate a new Question Type into the `@formbricks/surveys` package. This is a preact package, so check if it's suitable to use the `@calcom/embed-react` package. If you run into problems with that, please use the Javascript embed. Make sure that the Javascript embed code (`https://cal.com/embed.js`) is only loaded, when a Booking Question Type is rendered. Use a Loading indicator while the script is loaded.
* Test the new Integration in Link Surveys as well as web-app surveys (You can use our [demo app](https://formbricks.com/docs/contributing/demo) for that.

If you have Questions, please reach out. We also have an official [Formbricks Discord](https://formbricks.com/discord) for dev support.

**Comment by PeerRich** [comment]
we can also add formbricks to our app store and link to it once this is ready!

**Comment by PeerRich** [comment]
@mattinannt it would be cool if there was an example page on formbricks that explains how to use this feature once ready

**Comment by Ajay8302** [comment]
@PeerRich hi, I think example Is there under the create survey but It is not attractive and understandable and not looking good to according to the site theme

**Comment by Narottamchy** [comment]
I will try this 
/attempt #10811

**Comment by kremedev** [comment]
@PeerRich hey, i think claim didn't work here https://github.com/formbricks/formbricks/pull/708

/claim #10811

**Comment by RonankiDinesh** [comment]
I will try it 
/attempt #10811

**Comment by RonankiDinesh** [comment]
Can you please give some brief of what to do in this issue #10811

**Comment by ameeetgaikwad** [comment]
I would like to do this, what's the process?

**Comment by itsalam** [comment]
@PeerRich, I've noticed the `@calcom/embed-react` package doesn't have any type declarations, which may be troublesome for tsx imports as we would have here, would this require a new issue to solve for future uses?

**Comment by Nayaker** [comment]
I would like to do this, what's the process?

**Comment by atmanandsah** [comment]
Hi @PeerRich 
You want to embed the booking tab of cal.com into the survey page of formbricks.Is my understanding is correct please guide me. 

![Screenshot 2023-10-25 at 12 50 43 AM](https://github.com/calcom/cal.com/assets/12816307/9333c899-5afd-4865-8517-6454b83a3159)
<img width="1680" alt="Screenshot 2023-10-25 at 12 44 07 AM" src="https://github.com/calcom/cal.com/assets/12816307/b32cc670-8796-4f5b-b300-49741b0d0622">

**Comment by PeerRich** [comment]
thank you @KapadiaNaitik
